frontend/src/v5/ui/v4Adapter/overrides/cards/sharedStyles/sequencesTab.overrides.ts

```diff
@@ -35,11 +35,14 @@ export default css`
 		}
 		${SequenceDateContainer} {
 			width: 190px;
-			${SequenceDateField} {
+			${SequenceDateField} { // HERE
+				outline: 1px solid peru;
 				.MuiInputBase-root {
+					outline: 1px solid hotpink;
 					margin: 0px;
-
+					
 					input {
+						outline: 1px solid lightgoldenrodyellow;
 						color: ${({ theme }) => theme.palette.secondary.main};
```

Comment:
Beware of what the error state looks like
![image](https://github.com/3drepo/3drepo.io/assets/43547703/7937bb31-2e9c-43f8-8617-dfced6878230)

Reply:
@The-Daniel seems that the component is affecting the 4d tab in issues and risks.